lib/datadog/core/quantization/hash.rb

```diff
@@ -0,0 +1,106 @@
+# typed: true
+module Datadog
+  module Core
+    module Quantization
+      # Quantization for Hash
+      module Hash
```

Comment:
Have you considered moving this to contrib instead? It seems to be quite specific and only be used by integrations.

Reply:
Yeah, maybe we can move this into `Contrib::Utils`.